cli/videos-update/index.js

```diff
@@ -0,0 +1,127 @@
+import async from 'async'
+import chalk from 'chalk'
+import categoryServices
+  from '../../static/database/categories/services/index.json'
+import {
+  loadStaticJson,
+  writeStaticJson
+} from '../helpers/filesystem'
+import { isVideoMissingDetails, youtubeDurationToSeconds } from '../helpers/videos'
+import { flattenServicesList } from '../helpers/content'
+import { caseTitleToSnake } from '../../utils/text'
+import YoutubeClient from '../services/YoutubeClient'
+
+async function updateAllVideosDetails (categoryServices) {
+  if (!categoryServices) {
+    console.log(chalk.red('Services categories list is not found in "static/database/categories/services/index.json"'))
+    return
+  }
+
+  // collect all services in a single dimension array
+  const services = flattenServicesList(categoryServices)
+
+  return async.each(services, updateServicesVideosDetails)
+}
+
+/**
+ * Fetch and format details for each provided video
+ * @param unfilledVideos - list of videos to fetch details for
+ * @param serviceSlug
+ * @returns {Promise<unknown>}
+ */
+async function fetchVideosDetails (unfilledVideos, serviceSlug) {
+  return async.mapLimit(unfilledVideos, 1, async (video, callback) => {
+    try {
+      const videoDetails = await YoutubeClient.getVideoDetails('3xzhW-2aD4s')
```

Comment:
```suggestion
      const videoDetails = await YoutubeClient.getVideoDetails(video.videoId)
```

If you apply this fix, the CLI returns: 

```
Unable to load video details. Video is private or doesn't exist. Skipping undefined
GaxiosError: No filter selected. Expected one of: id, myRating, chart
    at Gaxios._request (/Users/davidgatti/Documents/GitHub/AVC/avc_website/node_modules/gaxios/src/gaxios.ts:117:15)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
```